svgr.config.js

```diff
@@ -7,6 +7,9 @@ module.exports = {
   typescript: true,
   svgo: true,
   prettier: true,
+  svgoConfig: {
+    plugins: [{ removeViewBox: false }],
+  },
```

Comment:
Decided to add this. Some illustrations, for example `empty-state-illustration.svg` was not displayed correctly on smaller devices. I think we should apply this plugin to all of our SVGs. You can read why here https://github.com/svg/svgo/pull/1461